### ISSUE TITLE
Table rows-per-page menu broken

### ISSUE THREAD

**Issue by vasild** [quasarframework/quasar#4168]
### Software version

Quasar: 1.0.0-beta.23
@quasar/app (v1+ only): v1.0.0-beta.26
OS: FreeBSD 12
Node: 12.2.0
NPM: 6.9.0
Browsers: Firefox 67.0
iOS:
Android:
Any other software related to your bug:

### JsFiddle

No need for jsfiddle, as the bug is observed also on the docs page: https://v1.quasar-framework.org/vue-components/table#Usage

### What did you get as the error?
Some tables have `Records per page: 5` at the bottom. Nothing changes visually when `5` is clicked. The console shows:
```
InternalError: "too much recursion"
    focus https://v1.quasar-framework.org/js/vendor.5793ed61.js:24
    showPopup https://v1.quasar-framework.org/js/vendor.5793ed61.js:24
    focus https://v1.quasar-framework.org/js/vendor.5793ed61.js:24
    showPopup https://v1.quasar-framework.org/js/vendor.5793ed61.js:24
...
```

### What were you expecting?
A menu should be displayed showing various options for `Records per page`, e.g. 5, 10, 20.

### What steps did you take, to get the error?
Just open https://v1.quasar-framework.org/vue-components/table#Usage and try to click `5` below some of the tables, next to `Records per page`

This is a regression in 1.0.0-beta.23 and the bug does not exist in 1.0.0-beta.22.
I confirm that commit f6ad9077 is the one that broke it - `f6ad9077~` is ok, `f6ad9077` is broken.

**Comment by vasild** [comment]
The bug is not fixed in the latest `dev` branch - e7c0aba4 still exhibits it.

**Comment by pdanpdan** [comment]
Can you please tell us what user agent you're using on the browser?

**Comment by rstoenescu** [comment]
@vasild Fix will be available in v1-rc.1. Can you confirm this?
And also, please tell us the user agent of your browser. Thank you.

**Comment by vasild** [comment]
Thanks for the quick response.

I confirm that the bug is fixed in the latest `dev` branch (02143345). I guess fixed by f1abe907.

My UA is `Mozilla/5.0 (X11; FreeBSD amd64; rv:67.0) Gecko/20100101 Firefox/67.0`. It is not anonymized, but the previous code in `Platform.js` was a bit lame - assuming that if the OS is not "Windows, Linux or Mac" then it does not exist, kind of.

Thanks!
